### ISSUE TITLE
[Fx89] Layout issues with ac_popup_megabar_title_and_url_two_lines tweak

### ISSUE THREAD

**Issue by EliSadaka** [Aris-t2/CustomCSSforFx#359]
When using the `ac_popup_megabar_title_and_url_two_lines` tweak, certain megabar results have the incorrect layout. The "Visit" and "Search with..." text is too close to the top line, and the "Switch to Tab" button is misaligned as well. I tried disabling the `ac_popup_megabar_compact_results` and `ac_popup_result_font_size` tweaks which I also have enabled but it didn't resolve the issue.

![Screenshot 2021-06-02 163059](https://user-images.githubusercontent.com/35779188/120549355-83aa1b80-c3c1-11eb-8838-245b8044c973.png)
![Screenshot 2021-06-02 163221](https://user-images.githubusercontent.com/35779188/120549370-886ecf80-c3c1-11eb-9622-6c8cb8a6cd56.png)
![Screenshot 2021-06-02 163257](https://user-images.githubusercontent.com/35779188/120549386-8c025680-c3c1-11eb-9c85-1dd2327fb67f.png)

System information
- latest CustomCSSforFx files (yes / no [no support, if 'no']): yes
- OS & OS version: Windows 10 Pro 19042.985
- OS theme (if not default): Dark
- Firefox version: 89.0
- Firefox theme / lw-theme: Dark
- This projects 'settings' the issue occurs with: ac_popup_megabar_title_and_url_two_lines

**Comment by Aris-t2** [comment]
This should be gone on current pre releses.

**Comment by EliSadaka** [comment]
I'm still experiencing the misaligned "Switch to Tab" element on the latest release. The other issues are now resolved though.

**Comment by Aris-t2** [comment]
I found what caused this. Fix will be on next release.

**Comment by Aris-t2** [comment]
Check [3.2.2](https://github.com/Aris-t2/CustomCSSforFx/releases/tag/3.2.2) release.

**Comment by EliSadaka** [comment]
Thanks, all the issues I mentioned before were fixed but I just found one more. The element for when a link is open on another synced device is having a similar issue as what the "Switch to Tab" button was having. Probably not a super common issue but I'm guessing it's a similar fix.

![Screenshot 2021-06-05 184620](https://user-images.githubusercontent.com/35779188/120907330-79d92000-c62e-11eb-8954-c5b9f380558d.png)
